appimage/AppRun

```diff
@@ -25,5 +25,5 @@ do
     fi
 done
 
-(PYTHONHOME="${APPDIR}/opt/python3.7" exec "/bin/bash" "${APPDIR}/usr/src/monkey_island/linux/run_appimage.sh")
+(PYTHONHOME="${APPDIR}/opt/python3.7" exec "${APPDIR}/opt/python3.7/bin/python3.7" "${APPDIR}/usr/src/monkey_island.py" $@)
 exit "$?"
```

Comment:
When will the `exit 0` occur? Is there a way to gracefully exit the island?

Reply:
Currently, I'm not sure how to get a 0 return code from Monkey Island. We should probably add a shutdown button to the UI.